nemo/collections/asr/models/label_models.py

```diff
@@ -307,15 +307,15 @@ def training_step(self, batch, batch_idx):
         logits, _ = self.forward(input_signal=audio_signal, input_signal_length=audio_signal_len)
         loss = self.loss(logits=logits, labels=labels)
 
-        self.log('loss', loss)
+        self.log('loss', loss, sync_dist=True)
         self.log('learning_rate', self._optimizer.param_groups[0]['lr'])
         self.log('global_step', self.trainer.global_step)
 
         self._accuracy(logits=logits, labels=labels)
         top_k = self._accuracy.compute()
         self._accuracy.reset()
         for i, top_i in enumerate(top_k):
-            self.log(f'training_batch_accuracy_top@{i}', top_i)
+            self.log(f'training_batch_accuracy_top@{i}', top_i, sync_dist=True)
```

Comment:
```suggestion
            self.log(f'training_batch_accuracy_top@{i}', top_i)
```

Removing this as the metric handles aggregation.